ServiceWebsite/ServiceWebsite.AcceptanceTests/Hooks/DataSetUpTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;

```

Comment:
Never seen a test for code in another test. This test will be calling real 3rd party services, E.g. Getting a real bearer token, seems a bit pointless having this test as if the real test fails, we will know why anyway from the test results on the server...

Reply:
Its just to verify that our data and client setup is working as expected. Its not testing the tests themselves.